lib/hammer_cli_csv/subscriptions.rb

```diff
@@ -42,6 +43,19 @@ def export
         end
       end
 
+      def export_manifest(csv, organization)
+        @api.resource(:subscriptions).call(:index, {
+            'per_page' => 999999,
+            'organization_id' => organization['id']
+        })['results'].each do |subscription|
+          next if subscription['product_id'].to_i != 0  # Red Hat subs do not have number SKU
```

Comment:
Your way is more efficient, but you could also do something like `subscription['product_id'].to_i.to_s == subscription['product_id'].to_s` which may be more understandable. Up to you if you want to change it, both ways work fine.